moped-etl/data-tracker-sync/data_tracker_sync.py

```diff
@@ -194,7 +194,7 @@ def main(args):
 
         # Update Moped project with Knack record ID of created record
         logger.info(
-            f"Updating Knack ID {knack_record_id} from Moped project {moped_project_id}"
+            f"Updating Moped project {moped_project_id} with Knack ID {knack_record_id}"
```

Comment:
This comment had the logic backwards. We are updating the Moped record with the ID from the newly created Knack record.